docs/manual/en/introduction/Installation.html

```diff
@@ -73,7 +73,8 @@ <h2>Install from CDN or static hosting</h2>
 		&lt;script type="importmap">
 		  {
 		    "imports": {
-		      "three": "https://unpkg.com/three@&lt;version&gt;/build/three.module.js"
+		      "three": "https://unpkg.com/three@&lt;version&gt;/build/three.module.js",
+                      "three/addons/": "https://unpkg.com/three@&lt;version&gt;/examples/jsm/"
```

Comment:
Please remove this line. It is not relevant for the below minimal code example.

Reply:
Done.